.github/workflows/scripts/slack_notifications/cve_notifier.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'time'
+require 'feedjira'
+require 'httparty'
+require_relative 'slack_notifier'
+
+class CveNotifier < SlackNotifier
+  def self.check_for_cves
+    ruby_sec_feed = 'https://rubysec.com/atom.xml'
+    feed = Feedjira.parse(HTTParty.get(ruby_sec_feed).body)
+    time = Time.now.utc
+    cycle = 24 * 60 * 60
+    feed.entries.each do |entry|
+      break if time - entry.updated > cycle
+
+      SlackNotifier.send_slack_message(cve_message(entry.title, entry.entry_id))
+    end
+  end
+
+  def self.cve_message(title, url)
+    {text: ":rotating_light: #{title}\n<#{url}|More info here>"}.to_json
+  end
+end
+
+# This file runs on a 24 hour cycle via slack_notifications.yml and sends Slack updates for Ruby CVEs.
```

Comment:
I find this comment very useful, but it's location to be a bit odd. Personally, I'd love to see this comment at the very top of the file so that I can immediately see what the file is about after first opening it. Then here, it'd be cool to have a comment explaining the conditional check, though that's optional given how common this pattern is for Rubyists.

Reply:
Agree the top of the file is much better. [Done](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/commits/6b044062a798dca30a0c6649b73684c31246f934).